monitoring/grafana/dashboards/rbd-details.json

```diff
@@ -168,21 +168,21 @@
           "expr": "irate(ceph_rbd_write_bytes{pool=\"$Pool\", image=\"$Image\"}[30s])",
           "format": "time_series",
           "intervalFactor": 1,
-          "legendFormat": "Read {{instance}}",
+          "legendFormat": "Write",
           "refId": "A"
         },
         {
           "expr": "irate(ceph_rbd_read_bytes{pool=\"$Pool\", image=\"$Image\"}[30s])",
           "format": "time_series",
           "intervalFactor": 1,
-          "legendFormat": "Write {{instance}}",
+          "legendFormat": "Read",
           "refId": "B"
         }
       ],
       "thresholds": [],
       "timeFrom": null,
       "timeShift": null,
-      "title": "IO Bytes per Second",
+      "title": "Throughput",

```

Comment:
Is the unit of this metric (bytes per second) displayed elsewhere? Or would it make sense to add it to the title?

Reply:
Yeah. On the left side of the panel: `MBs`.
![y](https://user-images.githubusercontent.com/1691518/83122972-faee0500-a106-11ea-9d0a-a4b23af8ed0b.png)

It's the default representation of the Data Rate unit in Grafana. We also use this in some existing panels.
![Screenshot_20200528_171351](https://user-images.githubusercontent.com/1691518/83123004-05a89a00-a107-11ea-9e23-d25afad60077.png)